packages/cdktf/test/__snapshots__/output.test.ts.snap

```diff
@@ -2,13 +2,6 @@
 
 exports[`boolean output 1`] = `
 "{
-  \\"//\\": {
-    \\"metadata\\": {
-      \\"version\\": \\"stubbed\\",
-      \\"stackName\\": \\"test\\",
-      \\"backend\\": \\"local\\"
-    }
-  },
```

Comment:
With the metadata now being removed (which I like!) – do we have some kind of test left, that checks for the metadata to exist correctly as expected in the JSON?

Reply:
Our integration tests still have the metadata, there we just read the plain file